architecture/additional_concepts/flannel.adoc

```diff
@@ -0,0 +1,32 @@
+[[architecture-additional-concepts-flannel]]
+= Flannel
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[architecture-additional-concepts-flannel-overview]]
+== Overview
+
+*flannel* is a virtual networking layer designed specifically for containers. 
+{product-title} can use it for networking containers instead of the default
+software-defined networking (SDN) components.
+
+[[architecture-additional-concepts-flannel-architecture]]
+== Architecture
+
+Each host within the network runs an agent called *flanneld*, which is
+responsibile for:
+
+- Managing a unique subnet on each host
+- Distributing IP addresses to each container on its host
+- Mapping routes from one container to another, even if on different hosts
+
+Each *flanneld* agent provides this infomation to a centralized *etcd* store so
+other agents on hosts can create an overlay network and route packets to
+any container within the *flannel* network.
```

Comment:
Do we want to provide a reason / justification for why someone would choose this SDN solution over the other 2 offerings [redhat/openshift-ovs-subnet or redhat/openshift-ovs-multitenant](https://docs.openshift.com/container-platform/3.3/install_config/install/advanced_install.html#configuring-cluster-variables)?